### DISCUSSION TITLE
Can we transmit audio and video data by the data channel of Fast-DDS?

### DISCUSSION THREAD

**Discussion by safrans** [eProsima/Fast-DDS#3062] [Q&A]
### Is there an already existing issue for this?

- [X] I have searched the existing issues

### Expected behavior

This isn't a bug, just a question about audio/video communication by Fast-DDS
If we don't use RTP/WebRTC and similar protocols to transmit audio and video data, we hope we can transmit audio and video data by the data channel of Fast-DDS with high performance and low latency.

### Current behavior

none

### Steps to reproduce

none

### Fast DDS version/commit

none

### Platform/Architecture

Other. Please specify in Additional context section.

### Transport layer

Default configuration, UDPv4 & SHM

### Additional context

Android platform

**Comment by safrans** [comment]
I see video streaming in the below url. If we use FastDDS, can we transfer HD camera data or audio and video data of the screen
cast ?
https://fast-dds.docs.eprosima.com/en/v2.7.1/fastdds/use_cases/large_data/large_data.html?highlight=audio#example-video-streaming

15.3.6.2. Example: Video streaming
In this scenario, the application transmits a video stream between a Publisher and a Subscriber, at 50 fps. In real-time audio or video transmissions, it is usually preferred to have a high stable datarate feed, even at the cost of losing some samples. Losing one or two samples per second at 50 fps is more acceptable than freezing the video waiting for the retransmission of lost samples. Therefore, in this case [BEST_EFFORT_RELIABILITY_QOS](https://fast-dds.docs.eprosima.com/en/v2.7.1/fastdds/api_reference/dds_pim/core/policy/reliabilityqospolicykind.html#_CPPv4N8eprosima7fastdds3dds24ReliabilityQosPolicyKind27BEST_EFFORT_RELIABILITY_QOSE) can be appropriate.

**Comment by JLBuenoLopez** [comment]
According to [Fast DDS Contributing guidelines](https://github.com/eProsima/Fast-DDS/blob/master/CONTRIBUTING.md#contribution-guidelines) questions are to be asked in the corresponding discussion forum so I am going to proceed and move the issue.
